plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/SubversionClientServiceImpl.java

```diff
@@ -237,27 +240,27 @@ public void cleanup(final String projectPath, final List<String> paths,
     @Override
     public void showConflicts(final String projectPath, final List<String> paths, final AsyncCallback<List<String>> callback) {
         status(projectPath, paths, "infinity",
-                false, // @param ignoreExternals whether or not to ignore externals (--ignore-externals)
-                false, // @param showIgnored whether or not to show ignored paths (--no-ignored)
-                false, // @param showUpdates whether or not to show repository updates (--show-updates)
-                false, // @param showUnversioned whether or not to show unversioned paths (--quiet)
-                false, // @param verbose whether or not to be verbose (--verbose)
-                new ArrayList<String>(),
-
-                new AsyncRequestCallback<CLIOutputResponse>(dtoUnmarshallerFactory.newUnmarshaller(CLIOutputResponse.class)) {
-                    @Override
-                    protected void onSuccess(CLIOutputResponse result) {
-                        if (result != null) {
-                            List<String> conflictsList = parseConflictsList(result.getOutput());
-                            callback.onSuccess(conflictsList);
-                        } else {
-                            callback.onFailure(new Exception("showConflicts : no SvnResponse."));
-                        }
-                    }
-
-                    @Override
-                    protected void onFailure(Throwable exception) {
-                        callback.onFailure(exception);
+               false, // @param ignoreExternals whether or not to ignore externals (--ignore-externals)
+               false, // @param showIgnored whether or not to show ignored paths (--no-ignored)
+               false, // @param showUpdates whether or not to show repository updates (--show-updates)
+               false, // @param showUnversioned whether or not to show unversioned paths (--quiet)
+               false, // @param verbose whether or not to be verbose (--verbose)
+               new ArrayList<String>(),
```

Comment:
This block of code doesn't related to my issue. Maybe we can use Collections class